docs/client/full-api/api/templates.md

```diff
@@ -41,6 +41,31 @@ Example:
 Now you can invoke this helper with `{{dstache}}foo}}` in the template defined
 with `<{{! }}template name="myTemplate">`.
 
+Helpers can accept positional and keyword arguments:
+
+```javascript
+Template.myTemplate.helpers({
+  displayName: function (firstName, lastName, kw) {
```

Comment:
I think the real problem here is that it's kind of a weird API, so we should try to be as explicit as possible about it to avoid confusion - I think `kw` as a variable name is inherently confusing